.github/workflows/deploy-docker.yml

```diff
@@ -0,0 +1,29 @@
+name: Push to Docker image to AWS ECR and deploy to AWS EC2
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
We might want to add:

```yaml
    branches:
      - main
```

so we only run this when merging PRs to `main`

Reply:
Changed to [trigger on a merged pull request](https://github.com/smokestacklightnin/osm/pull/9/commits/f828cfa86e7ade789e81b2dfe4cddd813801606e)

Also [added debug `push` trigger which will be removed before merge](https://github.com/smokestacklightnin/osm/pull/9/commits/7322231289f6113a00ea107ccd6c2f4192ce2b6d)